### PR TITLE
Better Compose in production docs

### PR DESCRIPTION
**Pull request by bfirsh**
The Compose/Swarm integration has been working really well for
users, so it seems pretty safe to remove the scary warnings about
it not being ready.

Signed-off-by: Ben Firshman <ben@firshman.co.uk>

### CODE REVIEW

**Comment by shin-** [comment]
LGTM!

**Comment by dnephin** [comment]
sounds good. cc @londoncalling for docs review

File: docs/production.md [hunk 1]
```diff
@@ -12,13 +12,17 @@ weight=22
 
 ## Using Compose in production
 
-> Compose is still primarily aimed at development and testing environments.
-> Compose may be used for smaller production deployments, but is probably
-> not yet suitable for larger deployments.
+When you define your app with Compose in development, you can really easily
+use this definition to run your application across different environments.
```
Comment (dnephin) [inline docs/production.md]: I would drop "really"

File: docs/production.md [hunk 2]
```diff
@@ -12,13 +12,17 @@ weight=22
...
 
-When deploying to production, you'll almost certainly want to make changes to
-your app configuration that are more appropriate to a live environment. These
-changes may include:
+The most basic way to deploy an application is to run it on a single server,
```
Comment (dnephin) [inline docs/production.md]: "The easiest" maybe ?

File: docs/production.md (hunk 1 above)
Comment (bfirsh) [inline docs/production.md]: Sounds good, thanks

File: docs/production.md (hunk 2 above)
Comment (bfirsh) [inline docs/production.md]: :+1:

File: docs/production.md [hunk 3]
```diff
@@ -73,8 +78,8 @@ commands will work with no further configuration.
 system, exposes the same API as a single Docker host, which means you can use
 Compose against a Swarm instance and run your apps across multiple hosts.
 
-Compose/Swarm integration is still in the experimental stage, but if you'd like
-to explore and experiment, check out the [integration guide](swarm.md).
+Read more about the Compose/Swarm integration on the
```
Comment (londoncalling) [inline docs/production.md]: Minor nitpick: change Read more about the Compose/Swarm **in** the Integration Guide (instead of "on"). Other than that, LGTM!

Comment (bfirsh) [inline docs/production.md]: oops - not a nit pick, an entirely valid grammatical error ;)

**Comment by dnephin** [comment]
LGTM

**Comment by londoncalling** [comment]
LGTM

**Pull request merged**